src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -319,7 +319,15 @@ function MoneyRequestView({
 
     const shouldShowMapOrReceipt = showMapAsImage || hasReceipt;
     const shouldShowReceiptEmptyState = !hasReceipt && !isInvoice && (canEditReceipt || isAdmin || isApprover);
-    const noticeTypeViolations = transactionViolations?.filter((violation) => violation.type === 'notice').map((v) => ViolationsUtils.getViolationTranslation(v, translate)) ?? [];
+    const receiptNoticeNames: OnyxTypes.ViolationName[] = [
```

Comment:
RECEIPT_REQUIRED is a violation with type=violation, and SMARTSCAN_FAILED has type=warning, so let's update this and other mentions of "notices" referring to these, to prevent confusion
```suggestion
    const receiptViolationNames: OnyxTypes.ViolationName[] = [
```